mscore/scoreitemmodel.h

```diff
@@ -0,0 +1,53 @@
+//=============================================================================
+//  ScoreItemModel
+//
+//  Copyright (C) 2020 Peter Jonas
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCOREITEMMODEL_H__
+#define __SCOREITEMMODEL_H__
+
+#include "libmscore/score.h"
+
+namespace Ms {
+class ScoreElement;
+class Score;
```

Comment:
You don't need to forward declare these since you have included `score.h`. You should include `scoreElement.h` since you have a function that returns a pointer of that type.